src/pages/detail-page/sections/mobile/MainSection.jsx

```diff
@@ -0,0 +1,59 @@
+import DonationInfo from '@components/DonationInfo';
+import MainTitle from '../../components/MainTitle';
+import Button from '@components/common/Button';
+
+const MainSection = () => {
+  return (
+    <div className='w-srceen flex h-screen flex-col items-center bg-black'>
+      <div className='relative h-[calc(100vh_-_50rem)]'>
+        <div className="h-[calc(100vh_-_50rem)] w-screen bg-[linear-gradient(to_right,rgba(0,0,0,0)_0%,rgba(0,0,0,0.8)_100%),url('https://img.news-wa.com/img/upload/2025/02/09/NWC_20250209182654.png.webp')] bg-cover bg-center"></div>
+        <div className='absolute bottom-0 h-full'>
+          <div className='inset-0 z-10 h-full w-screen bg-gradient-to-b from-transparent via-black/40 to-black'></div>
+        </div>
+        <div className='absolute bottom-0 left-[5vw]'>
+          <MainTitle title='1주년 기념 팝업 카페' name='KARINA' size='s' />
+        </div>
+      </div>
```

Comment:
`absolute bottom-0`과 같은 스타일을 여러 번 사용하고 있는데, 다음과 같이 효율적으로 표현할 수도 있을 거 같습니다.

```suggestion
      <div className="relative min-h-screen">
        <div className="absolute inset-0 bg-gradient-to-b from-transparent via-black/40 to-black z-10"></div>
        <div className="absolute bottom-0 left-[5vw] z-20">
          <MainTitle title='1주년 기념 팝업 카페' name='KARINA' size='s' />
        </div>
      </div>
```

Reply:
이 부분은 아이돌 이미지 위에 검은색 그라디언트 배경 레이어, 그 위에 제목 레이어를 까는 부분입니다.
말씀해주신대로 수정해보니 검은색 그라디언트 배경이 사라지네요....ㅠㅠ
하지만 모바일 디자인은 데이터 연결할 때 지금처럼 `MainSection + DetailSectioin`이 아니라, `MainSection` 하나만 사용하게끔 수정할 예정이라, 그때 다시 한 번 확인해보겠습니다!